Google.py

```diff
@@ -78,9 +80,18 @@ def __create_company(self, googlers):
 
         Args:
             googlers: An list of Googler objects.
-
         """
         #TODO: Implement function to create managerial hierarchy.
-        return
+
+        # Current implementation utilizes a dictionary to represent a pseudo company hierarchy.
+        # Every Googler is a key with its value being the Googler object that represents its manager.
+        company = {}
+        for googler in googlers:
+            # For every Googler, find their manager and map googler --> manager in dict
+            # NOTE: Believe this is implicitly O(n) making this algo O(n^2) :( if you're reading this guys, im using the thinker
+            manager = next((manager for manager in googlers if manager.id == googler.manager_id), None)
```

Comment:
This is super terse. I'll be honest I don't really know what is going on here. 

That being said I don't think the map should be <googler,googler> I think it should be <integer,googler> you could build it in o(n) just by iterating through the csv file. Alternativenly, if you want to take in a list of googlers, just map the google Id to the googler. The idea is that if the manager_id in the data maps to a user_id you can follow the chain that way. We can talk about it tomorrow in the standup.

Reply:
I see what you're saying...  I'll make the change because then it does become O(1)